src/pages/managerDashboard.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Link from 'next/link';
+import Image from 'next/image';
+import NavBar from '@/components/Header/NavBar';
+import FooterBar from '@/components/common/FooterBar';
+import DashboardListing from '@/components/singledashboardlisting/DashboardListing';
+// import styles from '../styles/managerDashboard.module.css';
+
+function managerDashboard () {
```

Comment:
@istoga I think we want the component name to be `PascalCase` - the first letter is uppercase. 🙏🏾

Reply:
My bad, will fix.

Reply:
Done